### ISSUE TITLE
Impaired self monitoring

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#5855]
**Preferred term label:**
Impaired self monitoring

**Definition (free text, please give PubMed ID)**
Impaired ability to track awareness of the effect that their behavior has on others and how it compares with standards or expectations for behavior.
Source: 
NCIT:C121466

**Parent term (use hpo.jax.org/app)**
'Behavioral abnormality'

This is needed for the Kids First INCLUDE supplement.

**Comment by pnrobinson** [comment]
added
